tests/utils-shared/src/main/java/org/keycloak/testsuite/util/oauth/AbstractOAuthClient.java

```diff
@@ -209,6 +218,39 @@ public AccessTokenResponse doTokenExchange(String subjectToken) {
         return tokenExchangeRequest(subjectToken).send();
     }
 
+    public AccessTokenResponse doFetchExternalIdpToken(String providerAlias, String accessToken) {
```

Comment:
I am not sure if `AbstractOAuthClient` is supposed to directly have methods like these on itself.

It seems the idea behind the recent refactoring of OAuthClient done by @stianst was, that `AbstractOAuthClient` just provides the methods for retrieve the "request" objects where the actual requests are invoked.

Could you please make this consistent with the existing stuff?

Reply:
Thanks, I reworked the code.